Gopkg.toml

```diff
@@ -69,7 +69,7 @@ ignored = ["github.com/Sirupsen/logrus", "github.com/gravitational/license/gener
 
 [[constraint]]
   name = "github.com/aws/aws-sdk-go"
-  version = "1.28.11"
+  version = "1.29.26"
```

Comment:
^ Version number needs to be bumped.

Reply:
I set it to the version from https://github.com/gravitational/teleport/issues/3510#issuecomment-607774810
Note that it's the _minimum_ version, the exact version is in `Gopkg.lock`.
Anyway, no reason not to make it higher so updated to match `Gopkg.lock`.

Reply:
Thanks for the clarification, I just scanned the PR earlier. Makes sense.